deployments.py

```diff
@@ -353,3 +354,23 @@ def _version() -> str:
             AwsEnv.labs: "0 0 * * *",  # Every day at midnight
         },
     )
+
+    # Vibe Check
+
+    create_deployment(
+        flow=vibe_check_inference,  # pyright: ignore[reportArgumentType]
+        description="Run vibe check inference on a set of concepts and push the results to s3. Optionally provide a custom list of Wikibase IDs to process.",
+        env_schedules={
+            # Once a week, at midday on Sunday
+            AwsEnv.labs: "0 12 * * SUN",
+        },
+    )
+
+    create_deployment(
+        flow=train_from_config,
+        description="Train classifiers for all concepts listed in vibe-checker/config.yml. Runs training in parallel and uploads results to Weights & Biases.",
+        gpu=False,
```

Comment:
in line with my other comment about naming, it seems like a great name for this would be `vibe_check_train`!